python/dgl/runtime/scheduler.py

```diff
@@ -146,14 +151,23 @@ def schedule_snr(graph,
     var_recv_nodes = var.IDX(recv_nodes, name='recv_nodes')
     # generate send and reduce schedule
     uv_getter = lambda: (var_u, var_v)
-    adj_creator = lambda: spmv.build_adj_matrix_uv((u, v), recv_nodes, graph.number_of_nodes())
-    inc_creator = lambda: spmv.build_inc_matrix_dst(v, recv_nodes)
```

Comment:
why remove incidence matrix? do we still support the inputs that message function is UDF and reduce function is a builtin?